src/applications/fry-dea/config/form.js

```diff
@@ -75,7 +79,7 @@ const formConfig = {
   trackingPrefix: 'fry-dea-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
-  formId: VA_FORM_IDS.FORM_22_5490,
+  // formId: VA_FORM_IDS.FORM_22_5490,
```

Comment:
Removing the formId will cause the build to fail. Please uncomment this line when committing.